mr/src/test/resources/log4j.properties

```diff
@@ -1,17 +1,3 @@
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
```

Comment:
I don't think this is the right license header. It should be the "Licensed to the Apache Software Foundation . . ." one.

Reply:
yes right, I've updated them with the correct ASF header